splunk_eventgen/__main__.py

```diff
@@ -53,6 +53,8 @@ def parse_args():
     generate_subparser.add_argument(
         "--generator-queue-size", type=int, default=500, help="the max queue size for the "
         "generator queue, timer object puts all the generator tasks into this queue, default max size is 500")
+    generate_subparser.add_argument("--disable-logging", action="store_true",
+                                    help="disable logging")
```

Comment:
Probably put the default value as False explicitly as some kind of document.

Reply:
agree. I tried to find a suitable place for this info but it seems all other sub-command does not have this kind of doc either.